apps/site/src/app/(main)/login/Login.tsx

```diff
@@ -1,25 +1,15 @@
+import { redirect } from "next/navigation";
 import getUserIdentity from "@/lib/utils/getUserIdentity";
 import LoginForm from "./components/LoginForm";
 
-import water from "@/assets/backgrounds/water.jpg";
-import { redirect } from "next/navigation";
-
 async function Login() {
 	const identity = await getUserIdentity();
 	if (identity.uid !== null) redirect("/portal");
 
 	return (
-		<div
-			className="min-h-screen flex flex-col items-center justify-center"
-			style={{ backgroundImage: `url("${water.src}")` }}
-		>
-			<h1
-				className="font-display text-5xl md:text-7xl mb-10"
-				style={{
-					textShadow: "0px 0px 20px rgba(255, 255, 255, 0.75)",
-				}}
-			>
-				Log In
+		<div className="min-h-screen flex flex-col items-center justify-center">
+			<h1 className="font-display text-3xl md:text-5xl mb-20">
+				Login to Portal
```

Comment:
Note: the application page redirects here for new users, so it may be confusing to see it saying this goes to the Portal when users are trying to apply.